rest_framework_simplejwt/tokens.py

```diff
@@ -94,10 +94,14 @@ def verify(self):
         self.check_exp()
 
         # Ensure token id is present
-        if api_settings.JTI_CLAIM not in self.payload:
+        if (
+            api_settings.JTI_CLAIM is not None
+            and api_settings.JTI_CLAIM not in self.payload
+        ):
             raise TokenError(_('Token has no id'))
 
-        self.verify_token_type()
+        if api_settings.TOKEN_TYPE_CLAIM is not None:
+            self.verify_token_type()
```

Comment:
At first this seemed strange, but I remembered it's Auth0. If possible, please leave a comment mentioning how this should only apply to outside services. Customization of tokens should still stride to use the TOKEN_TYPE_CLAIM setting. A test case would be great!